modfiles/ui/dialogs/matrix_dialog.lua

```diff
@@ -12,10 +23,10 @@ local function refresh_item_category(modal_data, type)
     end
 end
 
-local function create_item_category(modal_data, type)
+local function create_item_category(modal_data, type, label_arg)
     local flow_category = modal_data.modal_elements.content_frame.add{type="flow", direction="vertical"}
 
-    local label_title = flow_category.add{type="label", caption={"fp.matrix_" .. type .. "_items"}}
+    local label_title = flow_category.add{type="label", caption={"fp.matrix_" .. type .. "_items", label_arg}}
```

Comment:
The `label_arg` gets used in `matrix_constrained_items` in `config.cfg`. Speaking of which, right now it's set to `Choose __1__ free items` - should we make it `free item` when `__1__` is 1 since it's singular?